ansible/library/acl_capabilities_facts.py

```diff
@@ -9,6 +8,14 @@
 short_description: Retrieve ACL capability information
 '''
 
+# swsssdk will be deprecate after 202205
+use_swsssdk = True
+try:
+    import swsssdk
+except ImportError:
+    from swsscommon import swsscommon
```

Comment:
>from swsscommon import swsscommon [](http://example.com/codeflow?start=4&length=33)

Could you explore importing the classes? Then the code could be unified.
```
from swsscommon.swsscommon import SonicDBConfig, SonicV2Connector
```

Reply:
Fixed.